src/js/smooch.jsx

```diff
@@ -161,6 +164,14 @@ export class Smooch {
                 }
             });
         }).then((loginResponse) => {
+            observable.on('socket:connected', () => {
```

Comment:
I'd be OK with faye.js directly calling conversation-service.getConversation

faye.js is already playing the role of gluing faye events to redux dispatching, so it makes sense.

Not about cutting corners, it's about embracing redux :D